src/app/ubs-admin/components/ubs-admin-order-details-form/ubs-admin-order-details-form.component.html

```diff
@@ -0,0 +1,124 @@
+<div class="container">
+    <form class="form">
+      <div>
+        <h3 class="form__headline pl-3">Деталі замовлення</h3>
+      </div>
+      <div class="w-100 d-flex justify-content-between">
+        <label class="col-sm-3 form__label">Послуга</label>
+        <label class="col-sm-2 form__label">Об'єм</label>
+        <label class="col-sm-2 form__label">Вартість за одиницю</label>
+        <label class="col-sm-2 text-center form__label">Планована к-ть</label>
+        <label class="col-sm-2 text-center form__label">Підтверджена к-ть</label>
+        <label class="col-sm-1 text-center form__label"></label>
+      </div>
+      <hr class="divider"/>
+      <div class="main">
+        <ul class="w-100 p-0">
+          <li class="d-flex justify-content-between align-items-baseline py-2">
+            <span class="col-sm-3">Безпечні відходи</span>
+            <span class="col-sm-2">120 л</span>
+            <span class="col-sm-2">250 грн</span>
+            <div class="col-sm-2 d-flex justify-content-center form-group count">
+              <input class="shadow-none form-control input p-2" type="number" min="0" max="999" placeholder="1" disabled/>
```

Comment:
I believe it should be not a `placeholder="1"` but `value="1"`
And the same in other places below

Reply:
Fixed